src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -1053,5 +1062,30 @@ private void CompleteDatabaseUnloading(DocumentDatabase database)
 
             database.DatabaseShutdownCompleted.Set();
         }
+
+        private void CheckDatabasePathsIntersection(string databaseName, RavenConfiguration configuration)
+        {
+            using (_serverStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                var allDatabasesRecords = _serverStore.Cluster.GetAllDatabases(context);
+                var parentDir = configuration.Core.DataDirectory.FullPath;
+
+                foreach (var currRecord in allDatabasesRecords)
+                {
+                    if (currRecord.DatabaseName.Equals(databaseName))
+                        continue;
+
+                    var currConfiguration = CreateDatabaseConfiguration(currRecord.DatabaseName, ignoreDisabledDatabase: true,
+                        ignoreBeenDeleted: true, ignoreNotRelevant: true, databaseRecord: currRecord);
+
```

Comment:
Here, you need a `try`, to avoid errors in creating _another_ database configuration.
In that case, just ignore it.

Reply:
Done.